jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -528,6 +531,9 @@ frontend https-in
       <%- end -%>
     <%- end -%>
   <%- end -%>
+  <% if p("ha_proxy.expect_proxy") -%>
```

Comment:
add a check in the frontend spec test that this config line is output when you have the expected configuration.